src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -185,6 +185,21 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('auto_start', action='store_true', help='Start the streaming endpoint automatically after creating it.')
         c.argument('ips', nargs='+', arg_group='Access Control Support', help='Space-separated list of allowed IP addresses for access control. Use "" to clear existing list.')
 
+    with self.argument_context('ams streaming endpoint update') as c:
+        c.argument('tags', arg_type=tags_type)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+        c.argument('description', help='The streaming endpoint description.')
+        c.argument('description', help='The streaming endpoint description.')
+        c.argument('availability_set_name', help='AvailabilitySet name.')
+        c.argument('max_cache_age', help='Max cache age.')
+        c.argument('custom_host_names', nargs='+', help='The custom host names of the streaming endpoint.')
+        c.argument('cdn_provider', arg_group='CDN Support', help='The CDN provider name.')
```

Comment:
If you take a look on what I've done before and just merged, I also display available providers. Please try to do it similar to me or extract it to a higher-scope command/group.